code/modules/clothing/outfits/standard.dm

```diff
@@ -282,7 +282,7 @@
 	glasses = /obj/item/clothing/glasses/thermal/eyepatch
 	ears = /obj/item/radio/headset/headset_cent/commander
 	mask = /obj/item/clothing/mask/cigarette/cigar/havana
-	head = /obj/item/clothing/head/helmet/space/beret
+	head = /obj/item/clothing/head/hats/centhat
```

Comment:
Keep the special space-proof armour for the admirals

Reply:
I mean, admins said they didn't like the look.

May I make a snowflake subtype of the hat that's space proof? Or is there a better way

Reply:
I guess